package.json

```diff
@@ -14,8 +14,8 @@
     "clean": "rimraf ./lib ./types",
     "test": "jest",
     "lint": "tslint --project .",
-    "prettier:write": "prettier --write '{bin,src,test}/**/*.{js,ts}'",
-    "prettier": "prettier --check '{bin,src,test}/**/*.{js,ts}'",
+    "prettier:write": "prettier --write \"{bin,src,test}/**/*.{js,ts}\"",
```

Comment:
The single quotes break running this command on windows, so I had to change to double quotes